presentation/src/main/java/com/example/pinterbest/presentation/adapters/CreatorsAdapter.kt

```diff
@@ -0,0 +1,55 @@
+package com.example.pinterbest.presentation.adapters
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.example.pinterbest.presentation.R
+import com.example.pinterbest.presentation.databinding.ViewHolderCreatorBinding
+import com.example.pinterbest.presentation.models.ProfileViewData
+
+class CreatorsAdapter :
+    RecyclerView.Adapter<CreatorsAdapter.ViewHolder>() {
+    private var profiles = listOf<ProfileViewData>()
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        val binding = ViewHolderCreatorBinding.inflate(
+            LayoutInflater.from(parent.context),
+            parent,
+            false
+        )
+        return ViewHolder(binding)
+    }
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        holder.bind(profiles[position])
+    }
+
+    override fun getItemCount() = profiles.size
+
+    fun updateList(profilesList: List<ProfileViewData>) {
+        profiles = profilesList
+        notifyDataSetChanged()
```

Comment:
`notifyDataSetChanged` is considered as bad practice. It's much better to rewrite using DiffUtils